cmd/buildah/common.go

```diff
@@ -57,3 +59,27 @@ func openBuilder(store storage.Store, name string) (builder *buildah.Builder, er
 func openBuilders(store storage.Store) (builders []*buildah.Builder, err error) {
 	return buildah.OpenAllBuilders(store)
 }
+
+/****
+* Returns up to the first two names in the slice whose value starts with
+* the same value as the shortName.
+*****/
+func findShortName(names []string, shortName string) []string {
```

Comment:
No, I just tried to sleeze it through.  I was working on it the other night and decided to put the convenience function in play.  If you'd rather I do it separately I can pull it.  It's not being called/used yet.  I do need to bump the '> 0' up to '> 1' though, that was a cut/paste error while I was testing it.